src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -62,3 +66,22 @@ class OPL.KoComponents.ViewModels.OrificeCalculator
 
     @compressibilityCorrection = ko.observableArray _.values OPL.OrificeCalculator.Config.Dictionaries.CompressibilityCorrection
     @chosenCompressibilityCorrection  = ko.observable OPL.OrificeCalculator.Config.Dictionaries.CompressibilityCorrection.none
+    @compressibilityCorrectionValue = ko.observable(1)
+    @betaRatio = ko.computed =>
+      @orificeBoreDiameter() / @selectedPipeID()
+
+    @velocityOfApproach = ko.computed =>
+      Math.ceil((1 / Math.sqrt 1 - @betaRatio() ** 4) * 10 ** 2) / 10 ** 2
+
+    @flowRate = ko.computed =>
+      coeffDischarge = 0.6
```

Comment:
These should be constants defined in the beginning of the viewModel. makes it more clear for whoever comes next